### DISCUSSION TITLE
End of Life for Python=3.6

### DISCUSSION THREAD

**Discussion by mmcky** [executablebooks/meta#473] [General]
@chrisjsewell had posted this link at provides a nice calendar of `python` end of life dates

https://devguide.python.org/#status-of-python-branches

We should remove `python=3.6` across all repos in January, 2022

- [ ] myst-parser
- [ ] myst-nb
- [ ] jupyter-book

**Comment by choldgraf** [comment]
I'd actually be a fan of something slightly more aggressive, which would be to only "officially" support the 3 last major releases of Python. So right now we'd support python 3.9/8/7. 3.6 would already no longer be "official"

**Reply by mmcky** [reply]
I can get on board with that :-)
